bin/node/src/flags/metrics.rs

```diff
@@ -18,7 +18,7 @@ pub struct MetricsArgs {
     #[arg(long = "metrics.port", default_value = "9090", env = "KONA_NODE_METRICS_PORT")]
     pub port: u16,
     /// The ip address to use to emit prometheus metrics.
-    #[arg(long = "metrics.address", default_value = "0.0.0.0", env = "KONA_NODE_METRICS_ADDR")]
+    #[arg(long = "metrics.addr", default_value = "0.0.0.0", env = "KONA_NODE_METRICS_ADDR")]
```

Comment:
Looks like this change breaks the `test_metrics_args_listen_ip` test below. After fixing that up, should be good to merge.